docker/Dockerfile

```diff
@@ -11,6 +11,8 @@ RUN apt-get update \
 		autogen \
 		# Used by host-bison
 		autopoint \
+		# Used by the Linux kernel
+		bc \
```

Comment:
Why are we adding stuff used by the Linux kernel? We don't build the Linux kernel (?).

Reply:
That's right, Managarm does not do so. However, the `mlibc-lfs` project reuses the same rootfs, and they sure do build the Linux kernel. This is thus in preparation for the eventual move of `mlibc-lfs` to the Managarm org. It's also a small change which is why I included it.